redisvl/query/query.py

```diff
@@ -188,6 +188,7 @@ def __init__(
         dialect: int = 2,
         sort_by: Optional[str] = None,
         in_order: bool = False,
+        normalize_cosine_distance: bool = False,
```

Comment:
Added to VectorQuery and RangeVectorQuery class because vector distance does not apply to filter and count queries and therefore would not be part of base.

Reply:
I also named the field `normalize_cosine_distance` since it only applies to that distance metric not to all vector calculations.